client/question/question.html

```diff
@@ -10,9 +10,10 @@
     <script src="../../third_party/codemirror-5.19.0/mode/python/python.js"></script>
     <script src="../../third_party/codemirror-5.19.0/addon/edit/matchbrackets.js"></script>
     <script src="../../third_party/ui-codemirror-0.3.0/ui-codemirror.min.js"></script>
-    <script src="../shared/modules/config.js"></script>
+    <script src="../config/config.js"></script>
```

Comment:
Do you think we could group these imports by module? I.e. all the ../config stuff first, then one blank line, then all the ../data stuff, then one blank line, then all the other stuff? Slightly easier to grok that way, I think.